fast64_internal/oot/data/oot_getters.py

```diff
@@ -0,0 +1,37 @@
+from xml.etree.ElementTree import parse as parseXML, Element
+
+
+def getRoot(xmlPath: str) -> Element:
```

Comment:
I think this function should be named after "xml" or "parsing" or both or something

`parseXmlGetRoot` 4head